to.sh

```diff
@@ -13,6 +13,8 @@ qoc() { oc $@ > /dev/null 2>&1; }
 
 apply() {
   _oc apply -f manifests/ds.yaml
+  _oc apply -f manifests/mco-add-swap.yaml
+  _oc apply -f manifests/kubelet-confiuration.yaml
```

Comment:
@fabiand let me check this. The apply should have a specific order, first the failSwapOn flag should take place, and creating the swap afterwards, otherwise kubelet may not start (because of swap)

Reply:
@fabiand I've checked in OCP and indeed the sequence is problematic, we have to apply the kubelet configuration first and then to `oc wait` on the worker MCP to be in upgraded state again, and only then to apply the MC. By applying the MC the kubelet will be restarted, possbily because the `Before=crio-install.service`

Reply:
Hey, I've adjusted the rodering, please review